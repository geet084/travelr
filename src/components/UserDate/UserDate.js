import React from 'react';

export const UserDate = ({ days, hours, num}) => {
  return (
    <div>
      <p className="test">Time elapsed since {this.state.userDate}: </p>
      <p className="test">
        <span>{days} days and </span>
        <span> ~{hours} hours</span>
      </p>
      <p className="test">Approx. total miles you've gone in that time: </p>
      <p className="test user-dist">{num}</p>
    </div>
  )
}
export default UserDate;