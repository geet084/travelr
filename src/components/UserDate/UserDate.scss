.user-dist {
  text-align: center;
}