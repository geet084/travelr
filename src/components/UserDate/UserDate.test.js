import React from 'react';
import { shallow } from 'enzyme';
import { UserDate } from './UserDate';

describe('UserDate', () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      days: 3,
      hours: 7,
      num: 123456789,
    }
    wrapper = shallow(<UserDate {...mockProps}/>);
  });

  describe('initial state', () => {
    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
