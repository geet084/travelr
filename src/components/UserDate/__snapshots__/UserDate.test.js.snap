// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDate initial state should match snapshot 1`] = `
<div
  id="bob"
>
  <p
    className="test"
  >
    Time elapsed since 
    : 
  </p>
  <p
    className="test"
  >
    <span>
      3
       days and 
    </span>
    <span>
       ~
      7
       hours
    </span>
  </p>
  <p
    className="test"
  >
    Approx. total miles you've gone in that time: 
  </p>
  <p
    className="test user-dist"
  >
    123456789
  </p>
</div>
`;
